examples/advanced/cugraph/graphsage.py

```diff
@@ -0,0 +1,200 @@
+import argparse
```

Comment:
Did you run this script? If so, what performance number did you obtain?

Reply:
Yes, in terms of pure training time (not including dataloading), `SAGEConv` takes 2.5s per epoch, while `CuGraphSAGEConv` takes 2.0s, despite the overhead of coo-to-csc conversion. Test accuracy is also the same.